### PR TITLE
Distinguish debug and info logging

### LINKED ISSUE #140: Operator logs sensitive information on info level
The operator allows reading various configuration items via `ResourceRef` which is great. However, upon retrieving the configs from the Stack CR it logs all values on `info` causing secrets leakage.

## Expected behavior

If these configs need to be logged at all, either mask these values before doing so or increase logging verbosity to `debug`.

## Current behavior

Secrets leak on `info`. Also the operator default manifests set the level to `debug`, see https://github.com/pulumi/pulumi-kubernetes-operator/blob/master/deploy/yaml/operator.yaml#L23

## Steps to reproduce

1. Deploy operator
2. Deploy stack

```
apiVersion: pulumi.com/v1alpha1
kind: Stack
metadata:
  name: stack
  namespace: pulumi-operator
spec:
  commit: xzy
  envRefs:
    PULUMI_ACCESS_TOKEN:
      filesystem:
        path: /path/to/pulumi_api_token
      type: FS
...
```

3. Observe log emitting all stack properties on `info`.

### PR DESCRIPTION
**Pull request by viveklak**
Fixes #140

### CODE REVIEW

File: pkg/controller/stack/session_test.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package stack
 import (
 	"errors"
 	"fmt"
+	"github.com/pulumi/pulumi-kubernetes-operator/pkg/logging"
```
Comment (elsesiy) [inline pkg/controller/stack/session_test.go:6]: Given the existing ordering, this import should be on L12 if I'm not mistaken

File: pkg/controller/stack/stack_controller.go [hunk 2]
```diff
@@ -7,14 +7,15 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/pulumi/pulumi-kubernetes-operator/pkg/logging"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
```
Comment (elsesiy) [inline pkg/controller/stack/stack_controller.go:11]: Same comment as above, these are not stdlib imports

File: pkg/logging/logger.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
+package logging
```
Comment (elsesiy) [inline pkg/logging/logger.go:1]: I just noticed that none of the files have the preamble such as https://github.com/pulumi/pulumi/blob/master/sdk/go/auto/stack_test.go#L1

File: pkg/controller/stack/session_test.go (hunk 1 above)
Comment (viveklak) [reply pkg/controller/stack/session_test.go:6]: We typically adhere to `go fmt` which is admittedly a little weird about it. Still prefer it to `go imports` which does some arbitrary newline sprinkling.

File: pkg/logging/logger.go (hunk 3 above)
Comment (viveklak) [reply pkg/logging/logger.go:1]: Yeah we recently added a check for this in pulumictl https://github.com/pulumi/pulumictl/tree/master/cmd/pulumictl/copyright. This repo hasn't been hooked up to it yet. Will do that separately.

**Review by lblackstone — APPROVED** [review]

**Pull request merged**